src/types/steps.ts

```diff
@@ -5,6 +5,7 @@ const STEP_POA = 'poa'
 const STEP_FACE = 'face'
 const STEP_COMPLETE = 'complete'
 const STEP_AUTH = 'auth'
+const STEP_MOTION = 'motion'
```

Comment:
The step is called `ACTIVE_VIDEO_CAPTURE` in the Android SDK, so maybe I'll have to keep the code name here too for consistency.